src/web/src/pages/NotFound.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+    <div id="notfound">
+		<div class="notfound">
+			<div class="notfound-404">
+				<h1>Oops!</h1>
+				<h2>404 - The Page can't be found</h2>
+			</div>
+			<a href="/">Go To Homepage!</a>
+		</div>
+	</div>
+</template>
+
+<script>
+export default {
+    name: "NotFoundPage",
+    components: {},
+    props: {},
+    data() {},
+    methods: {},
+}
+</script>
+
+<style>
+* {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+}
+
+#notfound {
+  position: relative;
+  height: 100vh;
+}
+
+#notfound .notfound {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  -webkit-transform: translate(-50%, -50%);
+      -ms-transform: translate(-50%, -50%);
+          transform: translate(-50%, -50%);
+}
+
+.notfound {
+  max-width: 520px;
+  width: 100%;
+  line-height: 1.4;
+  text-align: center;
+}
+
+.notfound .notfound-404 {
+  position: relative;
+  height: 200px;
+  margin: 0px auto 20px;
+  z-index: -1;
+}
+
+.notfound .notfound-404 h1 {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 236px;
+  font-weight: 200;
+  margin: 0px;
+  color: #211b19;
```

Comment:
The *Oops* text has a relatively low contrast in dark mode.  
I think removing this `color` to follow text color definitions in `dark.scss` will resolve this problem.

Also, not sure about if `Montserrat` is a safe font face in systems.  It seemed that my device fell back to default sans-serif font...